test/WebJobs.Script.Tests/Diagnostics/LinuxAppServiceEventGeneratorTests.cs

```diff
@@ -45,13 +48,16 @@ public LinuxAppServiceEventGeneratorTests()
 
             var loggerFactoryMock = new Mock<LinuxAppServiceFileLoggerFactory>(MockBehavior.Strict);
             loggerFactoryMock.Setup(f => f.GetOrCreate(It.IsAny<string>())).Returns<string>(s => _loggers[s]);
+            loggerFactoryMock.Setup(f => f.GetOrCreateBackoff(It.IsAny<string>())).Returns<string>(s => _loggers[s]);
```

Comment:
@pragnagopa  with the latest updates to remove the cache of file loggers I can't replace this code in the unit tests. 
Prior to the changes, loggers were never directly referenced.  They were passed as an object from the cache into the WriteEvent function.  Since they were behind a dictionary they weren't directly referenced other than as a var. 

I don't really understand what I need to do to replace the file loggers with Mock File Loggers now that the cache is no longer present.

Reply:
@CooperLink - see https://github.com/Azure/azure-functions-host/pull/9070/files - had to introduce interfaces to make the code more unit test friendly , if looks good,

1. Merged the PR into this branch
2. Rebase your current PR on latest in dev
3. Verify new code has unit test coverage

Reply:
Rebased and verified.